src/core/kmod-setup.c

```diff
@@ -28,6 +28,14 @@
 #include "capability-util.h"
 #include "kmod-setup.h"
 #include "macro.h"
+#include "fd-util.h"
+#include <fts.h>
+
+static inline void fts_closep(FTS **p) {
+        if (*p)
+                fts_close(*p);
+}
+#define _cleanup_fts_close_ _cleanup_(fts_closep)
```

Comment:
we nowadays avoid these additional `#defines` for cleanup calls, and just use `_cleanup_()`, except in some special cases. The extra macro doesn't really make anything shorter after all...